charts/incubator/deemix/values.yaml

```diff
@@ -0,0 +1,36 @@
+image:
+  repository: tccr.io/truecharts/deemix
+  pullPolicy: IfNotPresent
+  tag: latest@sha256:61eda30552bca8c1fa1dd04f70adb20bf9d6ed56
```

Comment:
Ah another app with only `latest` tag available -.- (Not your fault ofc)
Looks like your sha is a bit short.

Also, you have to open a PR to truecharts/containers on `mirror` folder to add the image on our mirror
(only then `tccr.io/truecharts` will work!

Reply:
Nop, not yet at least.
That's why I'm telling you :)

Reply:
I _think_ this is right? https://github.com/truecharts/containers/pull/136